src/mlpack/bindings/python/print_doc_functions_impl.hpp

```diff
@@ -244,13 +251,15 @@ std::string ProgramCall(const std::string& programName, Args... args)
  * Given the name of a binding, print a program call assuming that all options
  * are specified.  The programName should not be the output of GetBindingName().
  */
-inline std::string ProgramCall(const std::string& programName)
+inline std::string ProgramCall(const std::string& programName) // TODO: here programName is the bindingName??
```

Comment:
Right, if you wanted, you could change `programName` to `bindingName` and it should be fine. :+1: